data_model/src/expression/tests.rs

```diff
@@ -0,0 +1,59 @@
+use super::*;
+
+mod display {
+    use super::*;
+
+    #[test]
+    fn simple_expr_should_not_use_parentheses() {
+        let left = Value::U32(5);
+        let right = Value::U32(10);
+        let mult = Multiply::new(left, right);
+
+        assert_eq!(mult.to_string(), "5*10");
+    }
+
+    #[test]
+    fn complex_expr_should_use_parentheses() {
+        let left = Add::new(Value::U32(12), Value::U32(8));
+        let right = Value::U32(10);
+        let div = Divide::new(left, right);
+
+        assert_eq!(div.to_string(), "(12 + 8)/10");
+    }
+
+    #[test]
+    fn raise_to_expr_should_use_stars() {
```

Comment:
This and next test seems a bit strange to me because we are testing trivial behavior.

Idea not necessary for this PR, we could try to evaluate resulting string and compare result with expected result, e.g.:
```rust
  let left = Add::new(Value::U32(12), Value::U32(8));
  let right = Value::U32(10);
  let div = Divide::new(left, right);

  assert_eq!(eval(div.to_string()), (12 + 8) / 10);
```

Reply:
Yeah, it's trivial, but that's what someone else can chagne. If he changes this he will get failing tests, which means, that this display behaviour is important and was done for a reason

Reply:
About evaluation tests I can't say if it's really worth it. @appetrosyan ?

Reply:
I think it's probably overkill

Reply:
These tests don't test anything. `Display` is meant for humans to see, and you can achieve the same coverage by adding `iroha_logger::debug` statements.

Reply:
Okay, I'll rollback the commit which introduced this tests